src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/util/BlackDuckIssueTrackerCallbackUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.synopsys.integration.alert.provider.blackduck.collector.builder.util;
+
+import java.util.Optional;
+
+import com.synopsys.integration.alert.common.message.model.ComponentItemCallbackInfo;
+import com.synopsys.integration.alert.provider.blackduck.BlackDuckProviderKey;
+import com.synopsys.integration.blackduck.api.generated.view.ProjectVersionComponentView;
+import com.synopsys.integration.blackduck.api.manual.enumeration.NotificationType;
+
+public final class BlackDuckIssueTrackerCallbackUtil {
```

Comment:
Is there a reason we want this class to use static methods rather than using the `@Component` annotation and autowire this in where needed and autowire in the BlackDuckProviderKey?